Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```
@@ -501,3 +501,26 @@
   parent: [ClothingOuterStorageFoldableBaseOpened, ClothingOuterCoatoldRobo]
   id: ClothingOuterCoatoldRoboOpened
   name: old roboticist lab coat
+
+#fbi addition
+- type: entity
+  parent: [ClothingOuterStorageBase, AllowSuitStorageClothing]
+  id: ClothingOuterFBIcoat
+  name: FBI coat
+  description: There a small label inside the coat which read "mary".
```

it's probably best not to tie the item to mary, but if we do the sentence should be "There's a small label inside the coat that reads "Mary."

---

it should just be untied to mary for general use
```suggestion
  description: A coat used by an old Earth enforcement agency. How did this get all the way up here?
```